src/resource/file.ts

```diff
@@ -139,6 +139,16 @@ type SaveFileOptions = WriteFileOptions & {
  * be rejected. You can initialise it first using [[createContainerAt]], or directly save the file
  * at the desired location using [[overwriteFile]].
  *
+ * This function is primarily useful if the current user does not access to change existing files in
```

Comment:
```suggestion
 * This function is primarily useful if the current user does not have access to change existing files in
```

Reply:
Incorporated this and the other suggestions in all three instances of this doc.